CMakeSettings.json

```diff
@@ -0,0 +1,45 @@
+{
+  // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
+  "configurations": [
+    {
+      "name": "x64-Debug",
+      "generator": "Visual Studio 15 2017 Win64",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
```

Comment:
Is `msvc_x64_x64` correct? I use `msvc_x64`.

Reply:
I think they added it after we started using it https://docs.microsoft.com/en-us/cpp/ide/cmake-tools-for-visual-cpp.

> msvc_x64_x64 | Compile for AMD64 using 64-bit tools